packages/client/hmi-client/src/workflow/ops/model-coupling/model-coupling-operation.ts

```diff
@@ -0,0 +1,22 @@
+import { Operation, WorkflowOperationTypes } from '@/types/workflow';
+
+export interface ModelCouplingState {
+	modelIds: string[];
```

Comment:
What is the point of this? Are these different from the inputs?
I do not see this being used anywhere

Reply:
Not being used - will clean up.